floodlight/io/datasets.py

```diff
@@ -799,3 +801,278 @@ def _download_matches_info(self) -> None:
                 )
                 with open(season_file, "wb") as binary_file:
                     binary_file.write(download_from_url(season_host_url))
+
+
+class IDSSEDataset:
+    """This dataset loads the accompanying data set from the *An integrated dataset of
+     synchronized spatiotemporal and event data in elite soccer* paper. [1]_
+
+    Upon instantiation, the class checks if the specified data already exists in the
+    repository's root ``.data``-folder, and will download the files to this folder if
+    not. The default setting is to load the first match from the dataset. However, any
+    individual match or the entire dataset (~2.4 GB) can be downloaded.
+
+    Parameters
+    ----------
+    dataset_dir_name: str, optional
+        Name of subdirectory where the dataset is stored within the root .data
+        directory. Defaults to 'dfl_dataset'.
```

Comment:
should stick to one name, IDSSE is more explicit than "dfl" which seems internal naming

Reply:
Thanks!